utils/m0crate-io-conf

```diff
@@ -37,15 +37,15 @@ first_profile_fid() {
 ##
 ## Sample output:
 ## ```
-## 0x7200000000000001:0x28 172.28.128.150@tcp:12345:4:1
-## 0x7200000000000001:0x2b 172.28.128.150@tcp:12345:4:2
+## 0x7200000000000001:0x2b inet:tcp:10.230.240.185@5001
+## 0x7200000000000001:0x2e inet:tcp:10.230.240.185@5002
 ## ```
 motr_clients() {
     consul kv get -recurse m0conf/nodes |
         # Sample input:
         # ```
-        # m0conf/nodes/cmu/processes/12/endpoint:172.28.128.150@tcp:12345:2:2
-        # m0conf/nodes/cmu/processes/12/services/addb2:37
+        # m0conf/nodes/localhost/processes/29/endpoint:inet:tcp:10.230.250.106@5001
```

Comment:
This can be different for different transports.
hctl bootstrap supports both, lnet and libfab transports. Thus, we need to support endpoints for both the transports. Kindly add a transport type option to the script, use it to parse the endpoints accordingly.

Reply:
Addressed.

Reply:
@supriyachavan4398, I don't think it is good idea to grep `inet` to check if its a libfab endpoint or not, why not pass transport_type as an argument to the script?

Reply:
Done.

Reply:
@mssawant, In PreMerge Jenkins Job how to identify transport_type  and pass transport_type as argument.

In PreMerge Job we are calling "m0crate-io-conf" from "test-boot1" file


```
do_io() {
    $HARE_DIR/libexec/m0crate-io-conf > _m0crate-io.yaml
    dd if=/dev/urandom of=/tmp/128M bs=1M count=128
    time m0crate -S _m0crate-io.yaml
}
```

Reply:
@pavankrishnat, libfab must be default transport_type mentioned in CDF and in m0crate-io-conf. If conf-store is available transport_type must be fetched from conf-store.